src/template_builder.py

```diff
@@ -83,7 +82,7 @@ def build_template(template_parts_list_file, template_file_path):
             if full_template.is_empty():
                 full_template = new_template_part
             else:
-                full_template = Template.add_template_part(full_template,
-                                                           new_template_part)
+                full_template = Template.add_template_part(
+                    full_template, new_template_part)
```

Comment:
I think it would be cleaner to call the function on the existing object:

full_template.add_template_part(new_template_part)

no?